process.md

```diff
@@ -0,0 +1,108 @@
+- **Parameter Change Documentation:** Keeping track of any changes made to the network's parameters, ensuring they're well-documented and justified.
+
+**2.2 Coordination Meetings**
+
+- **Weekly Internal Report every Thursday** : A routine meeting to discuss progress, challenges, and plan for the upcoming week. The meeting must be documented in a Google Docs Shared where all the team can be aligned regarding the insights from the team.
+
+**2.3 Community Management**
+
+- **Bridge Management** (_Platforms: Element, Discord - TBD_): Choosing and maintaining communication platforms for effective interaction with the community.
+- **Paseo's GitHub Organization Maintenance** : Overseeing the GitHub repository, ensuring it's up-to-date and well-organised.
+- **Element Channels** : Managing various channels with different topics, such as:
+  - **Public** : Public channel to share every update and news to the complete community
+  - **Tech** : Developer coordination channel and other Support channel for efficient communication and collaboration between them.
+  - **Management Channels** : Specialised channels for internal and core team discussions.
+
+**2.4 Social Media Management**
+
+- **X Account** : Engaging with the community, sharing updates, and announcements.
+- Other Social Media Updates (_Weekly. Optional_): Regular updates across different platforms to keep the community informed.
+
+**3. Monitoring**
+
+- Validator Monitoring (Service Level Agreements): Ensuring validators operate within agreed-upon standards and parameters.
+- Bot Development:
+  - Large Balance Monitoring: Keeping track of significant balance changes in the network.
+  - OpenGov Monitoring: Monitoring governance activities and proposals.
+
+**4. Runtime Management**
+
+- **Track, Add, Remove Pallets** : Managing the components of the runtime environment to ensure optimal performance.
+- **Runtime Upgrades** : Establishing a timeline (TBD) and process for upgrading the network.
+- **Pallet Configuration Analysis** : Assessing whether new or existing pallet configurations align with Paseo's requirements.
+
+**5. Runtime Upgrade Testing and Coordination**
+
+- **Local Dry-Runs with Live State** : Testing upgrades in a controlled environment to ensure stability.
+- **Backporting** : Applying new features or fixes from the mainnet to the testnet.
+- **Bricked Parachains Resolution** : Developing strategies to deal with malfunctioning parachains.
+- **XCM HRMP Channels Management** : Handling cross-consensus message passing channels.
+- **XCM Debugging for Parachain Support** : Troubleshooting and supporting parachain-related issues.
+- **Parachain Slot Assignment:** Managing the allocation of slots to parachains.
+- **Benchmarking** : Recalculate weights for every Runtime upgrade, taking into account different hardware configurations from Polkadot.
+- **Building Deterministic Runtime Artifacts** : Ensuring the runtime is predictable and repeatable.
+
```

Comment:
```suggestion
**Coretime:** Lead migration from parachain slots into coretime sales. Also once we have more details about coretime, a proper process for handling core assigments will be put in place.
```

Reply:
This :)

https://github.com/paritytech/polkadot-sdk/blob/e349fc9ef8354eea1bafc1040c20d6fe3189e1ec/polkadot/runtime/parachains/src/coretime/migration.rs#L208